SPEC.md

```diff
@@ -495,6 +495,9 @@ The ACE must provide a Metadata service on the address given to the applications
 
 Clients querying any of these endpoints MUST specify the `Metadata-Flavor: AppContainer` header.
 
+**NOTE**: Implementations can insert an authorization token into AC_METADATA_URL.
+For example, AC_METADATA_URL can be set to `https://10.0.0.1:8888/Y4vFeVZzKM2T9rwkpWHfqXuGsNjS6O5c` with the path portion acting as a token to uniquely and securely identify a pod.
```

Comment:
That's what SHOULD is for, right?

On Thu, Apr 30, 2015 at 3:31 PM, Eugene Yakubovich <notifications@github.com
> wrote:

> In SPEC.md <https://github.com/appc/spec/pull/327#discussion_r29476888>:
>
> > @@ -495,6 +495,9 @@ The ACE must provide a Metadata service on the address given to the applications
> >
> >  Clients querying any of these endpoints MUST specify the `Metadata-Flavor: AppContainer` header.
> >
> > +**NOTE**: Implementations can insert an authorization token into AC_METADATA_URL.
> > +For example, AC_METADATA_URL can be set to `https://10.0.0.1:8888/Y4vFeVZzKM2T9rwkpWHfqXuGsNjS6O5c` <https://10.0.0.1:8888/Y4vFeVZzKM2T9rwkpWHfqXuGsNjS6O5c> with the path portion acting as a token to uniquely and securely identify a pod.
>
> I guess. To be honest, I still feel uncomfortable with specifying level of
> entropy. The level of security is something that the implementation should
> decide on its own. It's immaterial for interop and prescribing something
> that's not actually required always makes me uneasy.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/appc/spec/pull/327/files#r29476888>.
>